src/main/java/docbuddy/users/persistence/DatastoreManager.java

```diff
@@ -3,12 +3,20 @@
 import com.google.cloud.datastore.Datastore;
 import com.google.cloud.datastore.DatastoreOptions;
 import lombok.Getter;
+import lombok.Setter;
+import org.springframework.stereotype.Repository;
 
 @Getter
+@Setter
+@Repository
 public class DatastoreManager {
     private Datastore datastoreClient;
 
     public DatastoreManager() {
         this.datastoreClient = DatastoreOptions.getDefaultInstance().getService();
     }
+
+    public DatastoreManager(Datastore client) {
```

Comment:
Shoul't we use Lombok @AllArgsConstructor?

Reply:
Good catch. Done.